[DM-3529] Create CL note (#2046)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y--deprecated-hookdevicegridaction-has-been-removed.md

---
date: ""
title: Deprecated hookDeviceGridAction has been removed

> product_area: Device management & connectivity

Reviewer: Should probably be Application enablement & solutions

Author: Changed.

change_type:
  - value: change-3BQrQ6adS

>     label: API change

Reviewer: Should be either Fix or Announcement

Author: Changed.

component:
  - value: component-YbYJ3gLU_

>     label: Web SDK

Reviewer: The component does not match the product area.

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3529
version: ""
---

> The deprecated hookDeviceGridAction has been removed. Instead the recently introduced hookDataGridActionControls should be used. It gives more control over data grid actions allowing actions to be added to any data grid, to override existing actions, etc.

Reviewer: ```suggestion
The deprecated `hookDeviceGridAction` has been removed. Instead the recently introduced `hookDataGridActionControls` should be used. It gives more control over data grid actions, such as allowing actions to be added to any data grid or to override existing actions.
```
